docs/developer-guide/ksqldb-rest-api/ksql-endpoint.md

```diff
@@ -30,6 +30,10 @@ SELECT statements use the `/query` endpoint.
 - **streamsProperties[``property-name``]** (string): The value of the property
 - named by ``property-name``. Both the value and ``property-name`` should be
   strings.
+- **sessionVariables** (map): Optional. Starting from 0.18, the parameter `sessionVariables` accepts
+  a map of variable names and values as initial variable substitution values. See

```

Comment:
Does this map have to have string keys and string values or is it more flexible than that? (We should clarify if so.)